content/en/tests/_index.md

```diff
@@ -103,6 +106,23 @@ The following tags are automatically collected to identify test configurations,
 | `ui.orientation`       | Orientation the UI is run in.                                   |
 | `ui.localization`      | Language of the application.                                    |
 
+### Parameterized test configurations
+
+When you run parameterized tests, the library detects and reports information about the parameters used.
+Parameters are a part of test configuration, so the same test case executed with different parameters will be considered as two different tests in Test Visibility.
+
+If a test parameter is non-deterministic and has a different value every time a test is run, each test execution will be considered a new test in Test Visibility.
+As a consequence, some product features may not work correctly for such tests: history of executions, flakiness detection, Intelligent Test Runner, and others.
+
+Some examples of non-deterministic test parameters are:
+- current date
+- a random value
+- a value that depends on the test execution environment (such as an absolute file path or the current username)
+- a value that has no deterministic string representation (for example an instance of a Java class whose `toString()` method is not overridden)
+
+It is best to try to avoid using non-deterministic test parameters.
+In case this is not possible, some testing frameworks provide a way to specify a deterministic string representation for a non-deterministic parameter (such as overriding parameter display name).
```

Comment:
```suggestion
Avoid using non-deterministic test parameters. In case this is not possible, some testing frameworks provide a way to specify a deterministic string representation for a non-deterministic parameter (such as overriding parameter display name).
```